src/EditorFeatures/Core/EditAndContinue/PdbMatchingSourceTextProvider.cs

```diff
@@ -29,7 +29,7 @@ internal sealed class PdbMatchingSourceTextProvider() : IEventListener, IPdbMatc
     private readonly object _guard = new();
 
     private bool _isActive;
-    private int _baselineSolutionVersion;
+    private int _baselineSolutionContentVersion;
```

Comment:
Renamed the generic 'SolutionVersion' concept in many placces to 'SolutionContentVersion' indicating that it revs as the content moves forward.